src/platform/primitives.js

```diff
@@ -0,0 +1,6 @@
+import styles from "styled-components"
+
+export const styled = {
+  Text: styles.span,
```

Comment:
@zephraph I just ran exogenesis locally, I see what you mean by a lot of classes. Looks like `react-native-web` is generating a separate class for every CSS property. But if that is the only issue, along with `TouchableOpacity` not working as intended. We should be able to fix those. For now tho, I think this is a fine solution if it works out.

Reply:
One thing that makes this _kinda_ ok is if you click into the react devtools tab in chrome this appears, which piggybacks on the ReactNative styled-components hook:

<img width="347" alt="screen shot 2018-06-07 at 11 44 49 am" src="https://user-images.githubusercontent.com/236943/41119660-33c941fc-6a48-11e8-8b84-4267f64e232d.png">